proto/nodeinstance/v1/connection.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+option go_package = "nodeinstance/v1;connection";
```

Comment:
should we follow `node`'s structure and have this like `nodeinstance/connection/v1;nodeinstanceconnection` ?

On that note, I've also opened https://github.com/netdata/aclk-schemas/pull/6 to fix the `_` inconsistency